src/Kentico.Xperience.Zapier/Admin/InfoModels/ApiKeyInfoProvider.cs

```diff
@@ -0,0 +1,17 @@
+using CMS.DataEngine;
+
+namespace Kentico.Integration.Zapier;
+
+/// <summary>
+/// Class providing <see cref="ApiKeyInfo"/> management.
+/// </summary>
+public partial class ApiKeyInfoProvider : AbstractInfoProvider<ApiKeyInfo, ApiKeyInfoProvider>
```

Comment:
Since we are injecting providers via IInfoProvider we should not need providers anymore. The only class we need per each data type is *Info.cs